src/Dofs/DofHandler.jl

```diff
@@ -108,6 +108,14 @@ mutable struct DofHandler{dim, G <: AbstractGrid{dim}} <: AbstractDofHandler
     ndofs::Int
 end
 
+function is_defined_on_full_domain(dh::DofHandler)
+    num_cells = 0
+    for sdh in dh.subdofhandlers
+        num_cells += length(sdh.cellset)
+    end
+    return getncells(get_grid(dh)) == num_cells
+end
+
```

Comment:
Wouldn't you have the same problem with a field that's defined on a subdomain, even if all cells are part of some `SubDofHandler`? In that case this check isn't sufficient, is it?

Reply:
Good catch, you are right. Let me fix that.